Scripts/script-ToTable.yml

```diff
@@ -5,7 +5,8 @@ commonfields:
 name: ToTable
 script: |-
   var headers = args.columns ? args.columns.split(',') : null;
-  return {Type: entryTypes.note, Contents: tableToMarkdown(args.title, args.data, headers), ContentsFormat: formats.markdown};
+  var flattenObject = treeToFlattenObject(args.data);
```

Comment:
`ToTable` receives an array and needs to flatten the elements while keeping an array to pass to `tableToMarkdown`